saga/sagaState.go

```diff
@@ -79,6 +88,19 @@ func (state *SagaState) IsTaskStarted(taskId string) bool {
 }
 
 /*
+ * Get Data Associated with Start Task, supplied as
+ * Part of the StartTask Message
+ */
+func (state *SagaState) GetStartTaskData(taskId string) []byte {
+	data, ok := state.taskData[taskId]
```

Comment:
You can return data directly right? The comment can indicate that it might be nil.